### PR TITLE
2022q3 two wireless reports

### PR DESCRIPTION
**Pull request by bzfbd**

### CODE REVIEW

File: 2022q3/wifi.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/~bz/wireless/[https://people.freebsd.org/~bz/wireless/]
+link:https://wiki.freebsd.org/WiFi/Rtw88[Realtek rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+link:https://wiki.freebsd.org/WiFi/Rtw89[Realtek rtw89 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw89[https://wiki.freebsd.org/WiFi/Rtw89]
+link:https://wiki.freebsd.org/WiFi/Mt76[MediaTek mt76 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Mt76[https://wiki.freebsd.org/WiFi/Mt76]
+link:https://wiki.freebsd.org/WiFi/Ath11k[QCA ath11k status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Ath11k[https://wiki.freebsd.org/WiFi/Ath11k]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+Currently development is mostly driven by Intel's iwlwifi driver again (see other report).
+As the saying goes ''each one helps the other'' so has work on Realtek's rtw89 driver helped to find a bug in LinuxKPI bothering iwlwifi users.
```
Comment (pauamma) [inline 2022q3/wifi.adoc:13]: s/helped to find/helped find/

File: 2022q3/wifi.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+For this status report the topic is mostly just on adding more drivers, which do need more LinuxKPI support.
```
Comment (pauamma) [inline 2022q3/wifi.adoc:14]: s/just on adding/

File: 2022q3/wifi.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+ * Realtek's rtw89 driver was committed to main but is not conected to the build yet.  Scanning already works but packets are not yet passing.  Having the driver in-tree already eased testing for users having that chipset in order to indentify more unimplemented LinuxKPI bits (some of which will help the other drivers as well) and reduced work for me.
+
+ * The next drivers to probably hit the tree will be based on MediaTek's mt76 driver (for 7921 and 7915) which I have compiling and started testing.
+
+ * Based on requests I am also working on ath11k to support STA mode given some vendors seem to ship Laptops with those chips.
+
+
+While some of this clearly benefits from work sponsored by the FreeBSD Foundation for iwlwifi and newer standard support, a lot of this is just free-time work.
+If you are interested in any of tese drivers I would greatly appreciate if some more hands would help with one or the other.
```
Comment (pauamma) [inline 2022q3/wifi.adoc:28]: s/tese/these/

**Review by pauamma — COMMENTED** [review]
I can either fix those myself on merge (they're small and not about content) or let you do it. If I don't hear from you in a few days, I'll do the former.

File: 2022q3/wifi.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ * Realtek's rtw88 PCI is in-tree as-is and after a fruitful discussion with Hans Petter at EuroBSDCon work on LinuxKPI USB support for the rtw88 USB WiFi dongles will continue.
+
+ * Realtek's rtw89 driver was committed to main but is not conected to the build yet.  Scanning already works but packets are not yet passing.  Having the driver in-tree already eased testing for users having that chipset in order to indentify more unimplemented LinuxKPI bits (some of which will help the other drivers as well) and reduced work for me.
+
+ * The next drivers to probably hit the tree will be based on MediaTek's mt76 driver (for 7921 and 7915) which I have compiling and started testing.
+
+ * Based on requests I am also working on ath11k to support STA mode given some vendors seem to ship Laptops with those chips.
+
+
+While some of this clearly benefits from work sponsored by the FreeBSD Foundation for iwlwifi and newer standard support, a lot of this is just free-time work.
```
Comment (grahamperrin) [inline 2022q3/wifi.adoc:27]: The FreeBSD Foundation

(uppercase T)

Oh, and thanks for all your free-time work!

File: 2022q3/wifi.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
+=== More wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/~bz/wireless/[https://people.freebsd.org/~bz/wireless/]
+link:https://wiki.freebsd.org/WiFi/Rtw88[Realtek rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+link:https://wiki.freebsd.org/WiFi/Rtw89[Realtek rtw89 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw89[https://wiki.freebsd.org/WiFi/Rtw89]
+link:https://wiki.freebsd.org/WiFi/Mt76[MediaTek mt76 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Mt76[https://wiki.freebsd.org/WiFi/Mt76]
+link:https://wiki.freebsd.org/WiFi/Ath11k[QCA ath11k status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Ath11k[https://wiki.freebsd.org/WiFi/Ath11k]
```
[the remark this answers is not in the document]
Comment (bzfbd) [reply 2022q3/wifi.adoc:8]: I have honestly no idea; that is beyond my markup knowledge

[the remark this answers is not in the document]
Comment (grahamperrin) [reply 2022q3/wifi.adoc:8]: <https://github.com/bzfbd/freebsd-quarterly/blob/2022q3/2022q3/wifi.adoc> previewed: 

image

As `Links: +` has the white space and `+` at the end of the line, the same can be done with the first four of the five _link:_ lines

**Review by pauamma — APPROVED** [review]
Could you avoid force-pushing next time? It makes re-reviewing harder.

**Comment by bzfbd** [comment]
@pauamma if you can tell me how to update the private branch and then the pull request, I'll be happy to change the workflow on github here.

**Comment by pauamma** [comment]
> @pauamma if you can tell me how to update the private branch and then the pull request, I'll be happy to change the workflow on github here.

If you push again (without forcing) from your local branch to your fork of freebsd/freebsd-quarterly after opening a pull request based on your first push, your pull request should have the 2nd push added to it automagically. Doing this lets me check the 2nd (or later) push against the changes I requested. So something like:
```
% git checkout main
% git pull ...
% git checkout -b 2022q3 (or whatever)
Write first draft.
% git add ...
% git commit ...
% git push ...
Open a pull request using the web interface. (URL for that is provided in answer to git push.)
Make changes following edit comments.
% git add ...
% git commit ...
% git push ... (no need to use --force)
Check the pull request, which should now have both commits in it.
```

**Comment by grahamperrin** [comment]
> … workflow on github here.

Prior to merge, you can simply: 

1. edit your copy of the file
2. commit. 

Visually, something like this: 

image

Note, the red arrow in the screenshot. 

If you aim to commit to a branch that is associated with a pull request, then you must:

* **browse to the branch** before editing. 

Visual orientation: 

image

* that was, the `freebsd-quarterly/2022q3/` directory in the `2022q3` branch in your fork
* for the current period, I guess that you'll have a `freebsd-quarterly/2022q4/` directory for your drafts.

HTH

**Pull request merged**